src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/orders.yml

```
@@ -352,3 +352,14 @@ admin_orders_display_customization_image:
     type: \d+
     name: \w+
     value: \w+
+
+admin_orders_product_prices:
+  path: /{orderId}/products/refresh-prices
```

This route is not refreshing prices, it's only fetching data from the database 🤔   if I am not wrong

```suggestion
  path: /{orderId}/products/get-prices
```

---

Indeed it had already been renamed in that sense (the controller function was also `refreshProductPricesAction`), but there must have been a rebase mistake or something